stdlib/public/SwiftRemoteMirror/SwiftRemoteMirror.cpp

```diff
@@ -423,11 +423,11 @@ static swift_childinfo_t convertChild(const TypeInfo *TI, unsigned Index) {
   };
 }
 
-static const char *convertError(SwiftReflectionContextRef ContextRef,
-                                llvm::Optional<std::string> Error) {
-  if (Error) {
-    ContextRef->lastError = *Error;
-    return ContextRef->lastError.c_str();
+static const char *convertString(SwiftReflectionContextRef ContextRef,
```

Comment:
Not sure if the name is ideal - it doesn’t really convert a trying, it gets the value.

Reply:
Agreed. Renamed to `returnableCString`.